kml/kml_tests/serdes_tests.cpp

```diff
@@ -849,8 +849,8 @@ UNIT_TEST(Kml_Deserialization_From_Bin_V8_And_V8MM)
     TEST(false, ("Exception raised", exc.what()));
   }
 
-  // Can't compare dataFromBinV8.m_categoryData and dataFromBinV8MM.m_categoryData directly
-  // because new format has less properties and different m_id. Compare some properties here:
+          // Can't compare dataFromBinV8.m_categoryData and dataFromBinV8MM.m_categoryData directly
+          // because new format has less properties and different m_id. Compare some properties here:
```

Comment:
```suggestion
// Can't compare dataFromBinV8.m_categoryData and dataFromBinV8MM.m_categoryData directly
// because new format has less properties and different m_id. Compare some properties here:
```